motr/idx_dix.c

```diff
@@ -1006,7 +1011,12 @@ static void dix_index_create_ast(struct m0_sm_group *grp, struct m0_sm_ast *ast)
 	M0_ENTRY();
 	dix_build(oi, &dix);
 	m0_clink_add(&dreq->dr_sm.sm_chan, &dix_req->idr_clink);
-	rc = m0_dix_create(dreq, &dix, 1, NULL, COF_CROW);
+	if (SKIP_CROW_IDX_OPS) {
```

Comment:
I think the log message is not that important here, change it to,
rc = m0_dix_create(dreq, &dix, 1, NULL, SKIP_CROW_IDX_OPS ? 0 : COF_CROW);